pyspark/bigdl/examples/lenet/lenet.py

```diff
@@ -0,0 +1,114 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from optparse import OptionParser
+from bigdl.dataset import mnist
+from bigdl.dataset.transformer import *
+from bigdl.nn.keras.layer import *
+from bigdl.nn.layer import Model
```

Comment:
remove nn.layer.Model?

Reply:
It is used in test for model loading https://github.com/intel-analytics/BigDL/pull/2296/files/c43ca0927cae8a33ff207aeeb67adaba55813354#diff-bbc826873e146fd0ef3de8f1cf5aa024R110
Or we can first remove test option. @zhichao-li